app/assets/v2/scss/base.scss

```diff
@@ -4,6 +4,14 @@
 html {
   --profile-step-fill: #cfbfff;
   --profile-step: var(--gc-blue);
+  scroll-behavior: smooth;
+}
+
+#btn-back-to-top {
+  position: fixed;
+  bottom: 20px;
+  right: 20px;
+  display: none;
```

Comment:
since the button element has been added at the beginning of the document, I noticed it's covered by other elements with position absolute, like the robot at the end of the `/bounties/funder` page. if we put something like `z-index: 9999` it fixes it.

Reply:
```suggestion
  display: none;
  z-index: 9999;
```